packages/ui/src/components/Loader/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import Proptypes from 'prop-types';
+
+import Dots from './dots';
+import Texts from './texts';
+
+const styles = StyleSheet.create({
+  container: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    height: '100%',
+    backgroundColor: '#fff',
+  },
+  contentLayout: { width: '60%' },
+});
+
+function Loader({ texts, ...props }) {
```

Comment:
This is great work. Don't hesitate tu put jsdoc on your function components or on hooks. :-)
You just need to write `/**` then press ENTER.